### PR TITLE
[audio|voice] Add console commands to troubleshoot audio sources and speech-to-text services

### PR DESCRIPTION
**Pull request by GiviMAD**
I would like add these new console commands to allow testing the audio sources and audio transcription capabilities.

Recently I was mentioned at https://community.openhab.org/t/problems-with-dialog-processing-with-pulseaudiobinding/155573 asking for help about these matters and it's difficult to give support about remote setups, so I though that having these commands can help users to troubleshot problems with the audio input and the transcription add-ons.

The new commands are:

### Voice Transcribe

Transcribe audio from default source, optionally specify a different source/file, speech-to-text service or locale.

#### Signature
```sh
voice transcribe [--source <source>]|[--file <file>] [<sttId>] [<locale>]
```
#### Example run:

```sh
openhab> voice transcribe --file turn_on_light.wav
Transcription: turn on light.
```

### Audio Record

Record a wav file of the specified seconds to the sounds folder.

#### Signature

```sh
audio record  [<source>] <seconds> <filename>
````

#### Example run:

```sh
openhab> audio record pulseaudio:source:df54bf45b3:Jabra_Speak2_40_Echo_Cancel_Source 5 turn_on_light
Record done.
```

### CODE REVIEW

File: bundles/org.openhab.core.voice/src/main/java/org/openhab/core/voice/internal/VoiceConsoleCommandExtension.java [hunk 1]
```diff
@@ -90,7 +98,10 @@ public VoiceConsoleCommandExtension(final @Reference VoiceManager voiceManager,
 
     @Override
     public List<String> getUsages() {
-        return List.of(buildCommandUsage(SUBCMD_SAY + " <text>", "speaks a text"),
+        return List.of(buildCommandUsage(SUBCMD_SAY + " <text>", "speaks a text"), buildCommandUsage(
+                SUBCMD_TRANSCRIBE + " [--source <source>]|[--file <file>] [<sttId>] [<locale>]",
+                "transcribe audio from default source, optionally specify a different source/file, speech-to-text service or locale"),
+                buildCommandUsage(SUBCMD_SAY + " <text>", "speaks a text"),
```
Comment (holgerfriedrich) [inline bundles/org.openhab.core.voice/src/main/java/org/openhab/core/voice/internal/VoiceConsoleCommandExtension.java:104]: this seems to be duplicate (see line 101)

File: bundles/org.openhab.core.voice/src/main/java/org/openhab/core/voice/internal/VoiceConsoleCommandExtension.java [hunk 2]
```diff
@@ -305,6 +320,51 @@ private void say(String[] args, Console console) {
...
+                return;
+            }
+            text = voiceManager.transcribe(fileAudioStream, parameters.get("sttId"), locale);
+        } else {
+            text = voiceManager.transcribe(parameters.get("source"), parameters.get("sttId"), null);
+        }
+        if (!text.isBlank()) {
+            console.println("Transcription: " + text);
+        } else {
+            console.println("No transcription generated.");
```
Comment (holgerfriedrich) [inline bundles/org.openhab.core.voice/src/main/java/org/openhab/core/voice/internal/VoiceConsoleCommandExtension.java:364]:
```suggestion
            console.println("No transcription generated");
```

**Review by holgerfriedrich — CHANGES_REQUESTED** [review]
Thank you!
Find a few suggestions regarding wording below, otherwise LGTM.

Would you mind creating a PR in the documentation repo?
I think it should be documented here:
https://github.com/openhab/openhab-docs/blob/main/configuration/multimedia.md

**Comment by GiviMAD** [comment]
I have to make a last moment commit to address an error on the transcribe description I saw while creating the docs PR.

Thank you for the review!

**Review by holgerfriedrich — APPROVED** [review]
LGTM, thank you

**Pull request merged**